### PR TITLE
simplify truffle-config, remove imports and reduce gas settings

### PR DESCRIPTION
**Pull request by poiresel**
Changes:

Removed rinkeby deploy from truffle-config.js
Removed any external imports in truffle-config.js to remove use of PrivateKeyProvider
Reduced gas settings for development and ganache to be within max gasLimit of default ganache-cli start

### CODE REVIEW

File: truffle-config.js [hunk 1]
```diff
@@ -13,24 +6,15 @@ module.exports = {
       port: 8545,
       network_id: '*', // Match any network id
       gasPrice: 10000000000,
-      gas: 7000000
```
Comment (alicevhenshaw) [inline truffle-config.js]: how come youve taken the gas down to 4800000?

Comment (alicevhenshaw) [reply truffle-config.js]: oh you answered this in your description

Comment (poiresel) [reply truffle-config.js]: 4,800,000 was an older default gas settings with a vanilla setup of ganache, however I can increase it to 6700000 which looks like the new default

Comment (alicevhenshaw) [reply truffle-config.js]: Yeah lets increase to 6.7m please 👍

**Pull request merged**